mk/xe-linux-distribution

```diff
@@ -150,6 +150,7 @@ identify_redhat()
                -e 's/^Enterprise Linux Enterprise Linux .* release \([0-9]*\)\.\([0-9]*\) (.*)$/distro=oracle;major=\1;minor=\2;/gp;' \
                -e 's/^Enterprise Linux Enterprise Linux .* release \([0-9]*\) (.*)$/distro=oracle;major=\1/gp;' \
                -e 's/^Oracle Linux Server release \([0-9]*\)\.\([0-9]*\)$/distro=oracle;major=\1;minor=\2/gp;' \
+               -e 's/^Sangoma Linux release \([0-9]*\)\.\([0-9]*\)\(\.[0-9]*\)\? (.*)/distro=centos;major=\1;minor=\2/gp;' \
```

Comment:
Is `Sangoma` distro = centos ? why not `distro = sangoma`?

Reply:
I believe it's because it's a modified CentOS distribution.

Reply:
I am not sure if a new verdor name should be added. The distro name is going to write into the xenstore.